src/test/java/org/jabref/logic/integrity/AbbreviationCheckerTest.java

```diff
@@ -32,4 +35,25 @@ void checkValueDoesNotComplainAboutJournalNameThatHasSameAbbreviation() {
         abbreviationRepository.addEntry(new Abbreviation("Journal", "Journal"));
         assertEquals(Optional.empty(), checker.checkValue("Journal"));
     }
+
+    @Test
+    void journalNameAcceptsFullForm() {
+        for (Field field : Arrays.asList(StandardField.BOOKTITLE, StandardField.JOURNAL)) {
+            IntegrityCheckTest.assertCorrect(IntegrityCheckTest.createContext(field, "IEEE Software"));
```

Comment:
Please use `checker.checkValue` instead of `IntegrityCheckTest...`. Reason: Keep the tests focused in the test subject.

See the name "AbbreviationCheckerTest"? It tests the class "AbbreviationChecker". This class is instantiated in line 25.

Also applies for the other checks you added. Could you please rework?

Reply:
@koppor I changed every class that has a checkValue method. But some of them like ASCIICharacterChecker.java has a different method(public List<IntegrityMessage> check(BibEntry entry)). Do you want me to change those tests too? If yes can you suggest me something?

Reply:
- No errors: `assertEquals(Collections.emptyList(), assciCharacterChecker.check(entry));`
- Errors: `assertEquals(List.of(new IntegrityCheckMessage("...text", StandardFields.DOI)), assciCharacterChecker.check(entry));)`. This way, the real error message is checked. And if someone changes something in the checker, it is also noted in the tests. See our long disucssions at https://github.com/JabRef/jabref/pull/6300 (which are also caused by failing tests - and we were forced to think about the intended behavior more deeply).